iati/about/models.py

```diff
@@ -1,8 +1,145 @@
 from django.db import models
 
-from wagtail.core.models import Page
+from wagtail.admin.edit_handlers import InlinePanel
+from wagtail.core.blocks import CharBlock, StreamBlock, StructBlock, TextBlock
+from wagtail.core.fields import StreamField
+from wagtail.core.models import Orderable
+from wagtail.documents.edit_handlers import DocumentChooserPanel
+from wagtail.images.blocks import ImageChooserBlock
+from wagtail.images.edit_handlers import ImageChooserPanel
 
-class AboutPage(Page):
-    pass
-    # parent_page_types = ['home.HomePage']
-    # subpage_types = ['home.AboutSubpage']
+from modelcluster.fields import ParentalKey
+from home.models import AbstractContentPage, AbstractIndexPage, IATIStreamBlock
+
+
+class AboutPage(AbstractContentPage):
+    """A model for the About landing page."""
+
+    parent_page_types = ['home.HomePage']
+    subpage_types = ['about.AboutSubPage', 'about.CaseStudyIndexPage', 'about.HistoryPage', 'about.PeoplePage']
+
+
+class AboutSubPage(AbstractContentPage):
+    """A model for generic About subpages."""
+
+    subpage_types = ['about.AboutSubPage']
+
+    multilingual_field_panels = [
+        InlinePanel('about_sub_page_documents', label='About subpage attachments'),
+    ]
+
+
+class AboutSubPageDocument(Orderable):
+    """A model for About sub page documents."""
+
+    page = ParentalKey(AboutSubPage, related_name='about_sub_page_documents')
+    document = models.ForeignKey(
+        'wagtaildocs.Document',
+        on_delete=models.CASCADE,
```

Comment:
Check consistency with other `on_delete` settings.

Reply:
After reading what these on_delete settings do, I think we should add to this `models.ForeignKey` and give it `null=True,blank=True` as well as `on_delete=models.SET_NULL`. Otherwise, I think as it is, if we deleted the attached document it would delete the AboutSubPage that references it.